photon-core/build.gradle

```diff
@@ -13,6 +13,10 @@ dependencies {
     implementation 'org.zeroturnaround:zt-zip:1.14'
 
     implementation "org.xerial:sqlite-jdbc:3.41.0.0"
+
+    def photonGlDriverLibVersion = "dev-v2023.1.0-6-g5e6f7fa"
```

Comment:
Move this to the ext {} block of the root build. Gradle alongside wpilib and stuff?